### PR TITLE
feat: add app recommender

### LINKED ISSUE #397: feat: add app recommender components
No design was prepared in figma.
An image is attached below for for inspiration purpose
798c56cc-6a02-419e-82af-51622bf30230

Make sure the font and color you used should match with website theme.

### PR DESCRIPTION
**Pull request by Shurtu-gal**
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #397 

### Post merge checklist

- [x] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [x] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

File: client/src/components/marginals/Footer.js [hunk 1]
```diff
@@ -302,6 +303,7 @@ const Footer = () => {
           </div>
         </div>
       </div>
+      <AppRecommender />
```
Comment (120EE0692) [inline client/src/components/marginals/Footer.js:306]: @Shurtu-gal for now comment this.

File: client/src/components/marginals/Footer.js [hunk 2]
```diff
@@ -10,6 +10,7 @@ import { ArrowRightCircle } from 'react-feather';
 import logo from '../../assets/images/logo.png';
 import googlePlay from '../../assets/images/logos/google_play.png';
 import { ARCHIVES } from '../../assets/placeholder/guide';
+import AppRecommender from '../widgets/AppRecomender';
```
Comment (120EE0692) [inline client/src/components/marginals/Footer.js:13]: @Shurtu-gal for now comment this.

**Review by 120EE0692 — CHANGES_REQUESTED** [review]

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
@Shurtu-gal run prettier in all files, in which you make changes.

File: client/src/assets/placeholder/appRecommender.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+export const APPS = Object.freeze({
+  welcomeMessage: 'Sent from Monday Morning App',
+  title: 'Continue with...',
+  sources: [
+    {
+      logo: blueLogo,
+      name: 'MM App',
+      current: false,
+      link: 'https://play.google.com/',
```
Comment (rutajdash) [inline client/src/assets/placeholder/appRecommender.js:15]: Please update this to the correct link.

File: client/src/assets/placeholder/appRecommender.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      name: 'Brave',
+      browser: 'Brave',
+      current: true,
+    },
+    {
+      logo: safari,
+      name: 'Safari',
+      browser: 'Safari',
+      current: true,
+    },
```
Comment (rutajdash) [inline client/src/assets/placeholder/appRecommender.js:40]: Comment out safari as we do not have and iOS or macOS app at the moment.

**Review by rutajdash — CHANGES_REQUESTED** [review]
1. Is there any way to check if the user already has the app installed?

2. Can we call on the app instead of a Play Store link? As far as I know, calling on an app to open in Android goes to the app or defaults to the same app's play store page. Find how it is done.

3. We also need to check OS before showing this instead of browser type because it should only appear on Android for now.

**Comment by rutajdash** [comment]
Check this out -
https://stackoverflow.com/questions/2958701/launch-custom-android-application-from-android-browser

File: client/src/assets/placeholder/appRecommender.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+      name: 'Edge',
+      browser: 'Microsoft Edge',
+      current: true,
+    },
+    {
+      logo: brave,
+      name: 'Brave',
+      browser: 'Brave',
+      current: true,
+    },
```
[the remark this answers is not in the document]
Comment (Shurtu-gal) [reply client/src/assets/placeholder/appRecommender.js:34]: Yeah, Chrome, Brave and Edge are the major browsers being used on Android devices currently.

File: client/src/assets/placeholder/appRecommender.js (hunk 4 above)
Comment (Shurtu-gal) [reply client/src/assets/placeholder/appRecommender.js:40]: Okay

File: client/src/assets/placeholder/appRecommender.js (hunk 3 above)
Comment (Shurtu-gal) [reply client/src/assets/placeholder/appRecommender.js:15]: Sure

**Comment by Shurtu-gal** [comment]
Currently, links to "https://mondaymorning.nitrkl.ac.in/article" are being redirected to the app.
But when it is not installed it is redirecting to the website.

**Comment by Shurtu-gal** [comment]
navigator.getInstalledRelatedApps() API can be used for checking if the app is installed.
It is supported on Chrome Android.

**Comment by rutajdash** [comment]
> navigator.getInstalledRelatedApps() API can be used for checking if the app is installed.
> It is supported on Chrome Android.

@120EE0692 
It would be good to implement this too I guess? Then we can call the app instead of going to playstore always.

**Comment by Shurtu-gal** [comment]
@rutajdash @120EE0692 is this still needed? Else it can be closed I guess.

**Comment by 120EE0692** [comment]
> @rutajdash @120EE0692 is this still needed? Else it can be closed I guess.

This is required.

**Comment by Shurtu-gal** [comment]
Closing this as `not-planned`.

**Pull request closed without merging**